web-registry/src/components/templates/MultiStep/MultiStepSection/StepperSection.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+
+import Stepper from 'web-components/lib/components/stepper';
+import OnBoardingSection from 'web-components/lib/components/section/OnBoardingSection';
+
+import { useMultiStep } from '../context/MultiStepContext';
+
+/**
+ *
+ * Stepper section
+ *
+ */
+
+type StepperSectionProps = {
+  children: JSX.Element | JSX.Element[];
+};
+
+export default function StepperSection({
```

Comment:
I see this was already merged, but let's try to avoid `export default` on components which aren't being lazy loaded see

https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad
https://rajeshnaroth.medium.com/avoid-es6-default-exports-a24142978a7a